src/redux/actions/deletetraine.ts

```diff
@@ -150,4 +152,71 @@ export const fetchtraine = ({ page,itemsPerPage,  All }:any) => {
                   dispatch({type: fetchtrainesss.fetchtraines_fail,error:err})})
         } catch(err:any) {toast.error(err.message);dispatch({type: fetchtrainesss.fetchtraines_fail,error:err})}
     }
-}
\ No newline at end of file
+}
+export const createtraine = ({ firstName, lastName, email, cycle_id }: any) => {
+    return async (dispatch: Dispatch<fetchact>) => {
```

Comment:
why have we created other actions?

Reply:
the first one is not being used on trainee page